cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -178,6 +178,20 @@ typedef enum {
     UART_FLAG_WAKEUP          = 0x2,    /**< wake from sleep on receive */
 } uart_flag_t;
 
+/**
+ * @brief   Available SERCOM UART data size selections
+ * @{
+ */
+#define HAVE_UART_DATA_BITS_T
+typedef enum {
+    UART_DATA_BITS_5 = 0x5,   /**< 5 data bits */
+    UART_DATA_BITS_6 = 0x6,   /**< 6 data bits */
+    UART_DATA_BITS_7 = 0x7,   /**< 7 data bits */
+    UART_DATA_BITS_8 = 0x0,   /**< 8 data bits */
+    UART_DATA_BITS_9 = 0x1,   /**< 9 data bits */
```

Comment:
I'm wondering if it would better to drop this line as the driver doesn't support it and it may be misleading for an user.
@aabadie

Reply:
I think it would break the API right? It should return a not supported.